quality_of_service_demo_py/quality_of_service_demo_py/liveliness.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import argparse
+
+from quality_of_service_demo_py.common_nodes import Listener
+from quality_of_service_demo_py.common_nodes import Talker
+
+import rclpy
+from rclpy.duration import Duration
+from rclpy.executors import SingleThreadedExecutor
+from rclpy.logging import get_logger
+from rclpy.qos import QoSLivelinessPolicy
+from rclpy.qos import QoSProfile
+from rclpy.qos_event import PublisherEventCallbacks
+from rclpy.qos_event import SubscriptionEventCallbacks
+
+POLICY_MAP = {
```

Comment:
I recently did something similar for reliability/durability in `ros2topic`: https://github.com/ros2/ros2cli/blob/a2d73c07f0245744e3d8e08d153752058470c9b4/ros2topic/ros2topic/api/__init__.py#L143

Would it make sense to centralize these in `rclpy`?

Reply:
It makes sense to me to centralize it. I can take a crack at that today.

Reply:
Note: I have created 
https://github.com/ros2/rclpy/pull/352
https://github.com/ros2/ros2cli/pull/240
to address this, but I won't change this PR until that has been agreed on. This will work as-is for the moment.

Reply:
We can apply this change his in a follow-up PR.